README.md

```diff
@@ -75,8 +76,7 @@ MongoDB, Express.js, Vue.js, Node.js, Mongoose ORM, Redis, Sendgrid, Nodemailer
 
 ## Contribution
 
-We are currently in the process of building test suites, docker images and writing extensive documentation to speed up startup time for developers. In the meantime, any Pull Requests/Issues found are appreciated.
-
+We are currently in the process of building test suites, and writing extensive documentation to speed up startup time for developers. In the meantime, any Pull Requests/Issues found are appreciated.
```

Comment:
Maybe a Quickstart section here, that points to the docker setup section of the documentation will be useful here for contributors.

Reply:
ah. that's what you meant.

Reply:
I'll add the quickstart at the top of the page.